app/ios/Podfile.lock

```diff
@@ -445,7 +445,7 @@ PODS:
   - React-jsinspector (0.72.5)
   - React-logger (0.72.5):
     - glog
-  - "react-native-attestation (1.0.0-alpha.330+a657da40)":
+  - react-native-attestation (1.0.0):
```

Comment:
I think this podfile needs to be updated again with the actual package rather than the linked version

Reply:
Fixed.